components/PageNavigation.vue

```diff
@@ -63,6 +63,7 @@
       color: $mediumgrey;
       text-decoration: none;
       text-transform: uppercase;
+      font-size: $font-size-extrasmall;
```

Comment:
Could you use `$font-size-small` instead? That is the size used in the design.

Reply:
sure I've changed to small